lib/src/app/permissions.dart

```diff
@@ -37,3 +51,45 @@ enum Permission {
     return name.replaceAllMapped(exp, (m) => '_${m.group(0)}').toLowerCase();
   }
 }
+
+class ViamAuthorization {
+  String authorizationType;
+
+  /// the authorization itself, either [AuthorizationId.organizationOwner] or [AuthorizationId.organizationOperator]
```

Comment:
[nit] can we make all these comments sentence case?

Reply:
Good call out, thanks! ✅